libr/core/rvc.c

```diff
@@ -479,9 +478,10 @@ R_API RList *r_vc_get_uncommitted(const char *rp) {
 		}
 		if (!found) {
 			if (!strcmp (NULLVAL, blob->fhash)) {
+				free (blob_absp);
 				continue;
 			}
-			char *fname = r_str_new (blob_absp) ;
+			char *fname = blob_absp;
```

Comment:
by reading the code above, i think blob_absp cant be null, so you dont need to check that if below. and as long as fname is only used in the list_append below its better to just pass the blob_basp to .append() instead of fname so you save one extra var